### ISSUE TITLE
Issue with screenshots for the failed tests

### ISSUE THREAD

**Issue by andromeda16** [damianszczepanik/cucumber-reporting#534]
Hi! I have experienced an issue with screenshots for the failed tests if screenshots look the same:
let's suppose we have 2 failed tests in a suite, 2 screenshots are done, size is the same and visually they look the same:
![target img](https://cloud.githubusercontent.com/assets/2178468/18946978/e71c294e-8639-11e6-93bb-2ce1a2874727.png)
Then in the Cucumber-JVM Report there will be an 'Attachment 1 (png)' link to a screenshot after each test, but only the 1st works, the 2nd link opens/collapses the 1st screenshot. Is it a feature or a bug?
HTML elements are exactly the same:
![screenshot div](https://cloud.githubusercontent.com/assets/2178468/18947063/34a88f36-863a-11e6-9ede-30676385c240.png)

**Comment by damianszczepanik** [comment]
Chech with the recent release because it should be fixed. If not attach json file so I can fix it.

**Comment by andromeda16** [comment]
Hi! Thanks much for the fast reply. 
Cucumber-JVM reports plug-in version: 2.8.0
[Json.txt](https://github.com/damianszczepanik/cucumber-reporting/files/500500/Json.txt)
*github doesn't allow to attach .json, so in .txt
Thanks much in advance.

**Comment by damianszczepanik** [comment]
The plugin is not supported so far for running in parallel. 

Duplicate of https://github.com/jenkinsci/cucumber-reports-plugin/issues/81
